src/styles/images/index.md.njk

```diff
@@ -25,10 +25,32 @@ If your image represents something physical, such as a letter, document or credi
 
 {{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
 
+
 ## Alt text
 
-All images must have an alt text. You should include a description of the image inside the tag to help users who are unable to see it.
+All images must have alt text. This is a description of an image that is read out by screen readers or is displayed if an image does not load or images have been switched off. The description tells people who cannot see an image what information the image provides.
+
+Alt text must:
+
+- describe the content and function of the image
+- be specific, meaningful and concise
+- use normal punctuation and end with a full stop
+
+Do not:
+
+- include the name of the photographer or person who created the image
+- start with ‘Image of’, ‘Graphic of’ or ‘Photo of’
+- repeat information from the page
+- include extra information not on the page
 
 {{ example({group: "styles", item: "images", example: "alt-text", html: true, open: true, size: "l"}) }}
 
-If your image is purely decorative and a description would be confusing you should still include the alt text, but leave it empty so it will be ignored by screen readers.
+So an image will be ignored by screen readers, use `alt=""` if it is:
```

Comment:
> screen readers

We could consider using the more broad term 'assistive technology' but granted we haven't really explained this term anywhere else.

(There has been some talk about having an accessibility page, which could explain this)

Reply:
@nickcolley I would stick with screen reader because it is not used by any other assistive technology. Looking at that sentence again, I would change it anyway.

Some images do not need descriptions. Use `alt=""` if the image is:
- decorative and does not include important content
- an icon that has a text label
- part of a link

What do you think?

Reply:
Nice 👍